README.md

```diff
@@ -1,29 +1,55 @@
 # GOV.UK CDN config
 
-Configuration for GOV.UK's content delivery network (CDN).
+Configuration for GOV.UK's content delivery network (CDN). You can read more about [how GOV.UK CDN works](https://docs.publishing.service.gov.uk/manual/cdn.html) in the developer docs.
 
-Configures the [Fastly CDN](https://fastly.com) from version-controllable VCL and
-YAML files, using Fastly's [API](https://docs.fastly.com/api/).
+⚠️ This repo has some [documented tech debt](https://trello.com/c/y6MIgxjp). It doesn't conform to GOV.UK standards, and lacks sufficient testing. Approach code changes with caution.
 
-## Fastly dictionaries
+## Tasks
+
+This repo contains 3 scripts to configure our [Fastly CDN](https://fastly.com) account.
+
+### Deploy Service
+
+Script: [deploy-service.sh](/blob/master/deploy-service.sh)
+
+Invoked via the [CDN: deploy service](https://deploy.publishing.service.gov.uk/job/Deploy_CDN) Jenkins job.
+
+This script allows you to configure a number of Fastly services:
+
+| service name | domain | description |
+| --- | --- | --- |
+| apt | apt.publishing.service.gov.uk | GOV.UK's Debian package repository |
+| assets | assets.publishing.service.gov.uk | the GOV.UK domain for uploads and static assets |
+| performanceplatform | service.gov.uk | the Performance Platform (https://www.gov.uk/performance) |
```

Comment:
Missing the full domain(s) for the performance platform service.

Reply:
👍 added